app/views/restaurants/_form.html.erb

```diff
@@ -1,65 +1,65 @@
-  </core-field>
-  <core-field>
-    <paper-input label="Phone Number" type="number" error="Please insert a phone number" layout vertical id="phone_number" required value="<%= @restaurant.phone_number %>"></paper-input>
-  </core-field>
-  <core-field>
-    <paper-input label="Email" type="email" error="Please insert a valid email" layout vertical id="email" value="<%= @restaurant.email %>"></paper-input>
-  </core-field>
-  <core-field>
-    <paper-input multiline label="Address" layout vertical id="address" value="<%= @restaurant.address %>"></paper-input>
-  </core-field>
-  <core-field>
-    <paper-input label="Twitter" type="text" layout vertical id="twitter" value="<%= @restaurant.twitter %>"></paper-input>
-  </core-field>
-  <core-field>
-    <paper-input label="Instagram" type="text" layout vertical id="instagram" value="<%= @restaurant.instagram %>"></paper-input>
-  </core-field>
-  <core-field>
-    <paper-input label="Facebook" type="text" layout vertical id="facebook" value="<%= @restaurant.facebook %>"></paper-input>
-  </core-field>
-  <core-field>
-    <paper-input label="Website" type="text" layout vertical id="website" value="<%= @restaurant.website %>"></paper-input>
-  </core-field>
-  <paper-checkbox label="Delivery" id="delivery" checked="<%= @restaurant.delivery %>"></paper-checkbox>
-
-  <div>
-    <a class="submit" %>
-      <div class="button raised primary">
-        <div class="center" fit>
-          <% if @restaurant.new_record? %>
-            Create
-          <% else %>
-            Update
-          <% end %>
-        </div>
-        <paper-ripple fit></paper-ripple>
-      </div>
-    </a>
+  <div class="field">
+    <%= f.label :image %><br>
+    <%= f.file_field :image %>
+  </div>
+  <div class="field">
+    <%= f.label :name %><br>
+    <%= f.text_field :name %>
+  </div>
+  <div class="field">
+    <%= f.label :description %><br>
+    <%= f.text_area :description %>
+  </div>
+  <div class="field">
+    <%= f.label :slogan %><br>
+    <%= f.text_field :slogan %>
+  </div>
+  <div class="field">
+    <%= f.label :phone_number %><br>
+    <%= f.text_field :phone_number %>
+  </div>
+  <div class="field">
+    <%= f.label :email %><br>
+    <%= f.text_field :email %>
+  </div>
+  <div class="field">
+    <%= f.label :address %><br>
+    <%= f.text_area :address %>
+  </div>
+  <div class="field">
+    <%= f.label :twitter %><br>
+    <%= f.text_field :twitter %>
+  </div>
+  <div class="field">
+    <%= f.label :instagram %><br>
+    <%= f.text_field :instagram %>
+  </div>
+  <div class="field">
+    <%= f.label :facebook %><br>
+    <%= f.text_field :facebook %>
+  </div>
+  <div class="field">
+    <%= f.label :website %><br>
+    <%= f.text_field :website %>
+  </div>
+  <div class="field">
+    <%= f.label :delivery %><br>
+    <%= f.check_box :delivery %>
+  </div>
+  <div class="actions">
+    <%= f.submit %>
   </div>
```

Comment:
After submit.... some code are missing...
-      <div class="button raised primary">		+    <%= f.label :facebook %><br>
-        <div class="center" fit>		+    <%= f.text_field :facebook %>
-          <% if @restaurant.new_record? %>		+  </div>
-            Create		+  <div class="field">
-          <% else %>		+    <%= f.label :website %><br>
-            Update		+    <%= f.text_field :website %>
-          <% end %>		+  </div>
-        </div>
Add it without polymers.